shoplist/src/main/java/com/netguru/codereview/network/ShopListApiMock.kt

```diff
@@ -8,20 +8,20 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.flow
 
 class ShopListApiMock : ShopListApi {
-    override suspend fun getShopLists(): List<ShopListResponse> =
+    override suspend fun getShopLists(): List<ShopListResponse?> =
```

Comment:
add Optional to the response of APIs, despite this being mocked, but when scale this code, it will be used as not-null, this could lead to error, prone, do not trust backend response.